server/models/group.js

```diff
@@ -119,6 +119,20 @@ GroupSchema.statics = {
     });
   },
 
+  search(name) {
```

Comment:
We call this method `search` but the only possible query parameter is name. I'd prefer it if we explicitly call the method `searchByName` or something similar so it's clear when it's used that this is only searching by name